src/poliastro/bodies.py

```diff
@@ -133,9 +133,9 @@ class _Mercury(_Body):
 
     @staticmethod
     def _rot_elements_at_epoch(T, d):
-        ra = (281.01 - 0.033 * T) * u.deg
-        dec = (61.45 - 0.005 * T) * u.deg
-        W = (329.548 + 6.1385025 * d) * u.deg
+        ra = (281.0097 - 0.0328 * T) * u.deg
```

Comment:
Can you please write a link at the top of the file to see where can we get these values from?

Reply:
Sure

Reply:
I think this was not done yet? We should have this information in the code, not only in the tests. Just a line in the top docstring stating where do they come from, as it's done in other modules.